code/modules/events/destructible_terrain.dm

```diff
@@ -0,0 +1,11 @@
+//Enables destructible terrain on the ground level
+/proc/enable_destructible_terrain()
+	var/list/turfs = get_all_of_type(/turf/closed/wall, TRUE)
+	for(var/turf/closed/wall/turf_for_editing in turfs)
+		if(turf_for_editing.z != 3)
+			//We only want to edit ground turfs!
+			continue
+		if(istype(turf_for_editing, /turf/closed/shuttle))
```

Comment:
an istype in this will be VERY intensive

Reply:
Would you rather me check the name of the turf or something else?

I moved the check to only check turfs that are `hull` so it should be a little bit less costly...